File: client/src/types/PeakFormValue.ts
export type PeakFormValue = {
  isPositive: boolean;
  isNegative: boolean;
  distance: number;
  threshold: number;
  isLoop: boolean;
  baseCh: number | undefined;
};

export const initPeakFormValue: PeakFormValue = {
  isPositive: false,
  isNegative: true,
  distance: 3000,
  threshold: 3,
  isLoop: false,
  baseCh: undefined,
};

File: server/model/peak_form_value.py
class PeakFormValue:
    def __init__(self, json_data) -> None:
        peak_form_value = json_data.get("peakFormValue")
        if peak_form_value:
            self.isPositive = peak_form_value.get("isPositive")
            self.isNegative = peak_form_value.get("isNegative")
            self.distance = peak_form_value.get("distance")
            self.threshold = peak_form_value.get("threshold")
            self.isLoop = peak_form_value.get("isLoop")
            self.base_ch = peak_form_value.get("baseCh")
